s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3SinkSpec.scala

```diff
@@ -41,7 +55,7 @@ class S3SinkSpec extends S3WireMockBase with S3ClientIntegrationSpec {
 
   it should "upload a stream of bytes to S3 with custom headers" in {
 
-    mockUpload()
+    //mockUpload()
```

Comment:
And this is obsolete?

Reply:
hmm... probably a left over for me trying to figure out why my tests didn't succeed initially... will revert